### DISCUSSION TITLE
AutoLayoutについて

### DISCUSSION THREAD

**Discussion by atsujr** [lit-iphone-course/lit-iphone-course#80] [Question-original] [answered]
(1) 実現したいこと
AutoLayoutをつけた際に、EditViewControllerでpreview(画像１枚目)だとうまく反映されているのに実機(画像2枚目)で実行してみると、完全にtextfieldが潰れてしまっている。
(2) 実現するために自分が試した内容とその結果
autolayoutを、AddViewControllerと同じようにつけてみたが、うまくいかなかった。
(3) 調べた内容や資料
AddViewControllerのAutolayoutはうまくいっていたので、そちらを参考に行った。
(4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
textFieldに対する制約が間違えている、、、？また、画像３枚目より、imageVIewが不自然に広がってしまっているが、制約を確認したところ、間違えている箇所は発見できなかった。
![スクリーンショット 2022-09-29 23 03 27](https://user-images.githubusercontent.com/105122292/193053000-57dd8c22-33ef-49a3-8a08-ae3b044efa91.png)
![IMG_8570](https://user-images.githubusercontent.com/105122292/193053152-a2907104-c150-4685-8c99-3a1d4955392c.PNG)
![スクリーンショット 2022-09-29 23 05 26](https://user-images.githubusercontent.com/105122292/193053489-ae092560-cde9-4827-951d-6959f13c1571.png)

**Comment by txxxxc** [comment, marked as answer]
Discordで解決済み
